src/NHibernate.Test/NHSpecificTest/GH3046/Fixture.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH3046
+{
+	// Duplicated from GH2549 with minor changes to trigger the trouble.
```

Comment:
Maybe better to collapse test into GH2549?

Reply:
Like 9f99166?

(Or maybe by defining the filter by code in `OnSetup` and changing its definition for #3046, but it was just easier to do the way I have done it.)

Reply:
I was thinking just adding a new test to the #2549 tests.

Reply:
It seemed to me simpler to parameterize them (0420362).

Reply:
Perfect